app/forms/nomination_form.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class NominationForm
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+
+  attr_accessor :local_authority_id, :school_id
+  validates :local_authority_id,
+            presence: { message: "We could not find any local authorities matching your search criteria" },
+            on: %i[local_authority save]
+  validates :school_id,
+            presence: { message: "We could not find any establishments matching your search criteria" },
+            on: %i[school save]
+
+  def attributes
+    { local_authority_id: nil, school_id: nil }
+  end
+
+  def available_schools
+    SchoolLocalAuthority.where(local_authority_id: local_authority_id).joins(:school).includes(:school).map(&:school)
```

Comment:
What's the biggest number this could be?

Reply:
787.
```
[787, "Kent"]
[784, "Lancashire"]
[685, "Hampshire"]
[672, "Hertfordshire"]
[646, "Essex"]
```
Are top 5.

Reply:
Oof... Ok for now I guess